content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -292,7 +292,9 @@ If the Pod [restarts](#pod-restart-reasons), or is restarted, all init container
 must execute again.
 
 Changes to the init container spec are limited to the container image field.
-Altering an init container image field is equivalent to restarting the Pod.
+Altering an init container image will not automatically trigger the Pod to restart.
+You can use a [workload controller](/docs/concepts/workloads/) to cause such updates
+to restart the affected pods.
```

Comment:
This new text changes the meaning around what is being changed; “altering an init container image” could be understood as meaning a push to an OCI registry.

Try this:
```suggestion
Directly altering a Pod to change the container image for an init container does **not**
automatically trigger the Pod to restart it containers.
You can use a [workload management](/docs/concepts/workloads/controllers/) object, with
implicit rollouts for changes. For example, if you change a Pod template embedded in
a Deployment, and you use a different container image for an init container,
the indirect change **does** trigger a rollout (Kubernetes makes new Pods based on the
updated template).
```

Reply:
Thank you, this is much clearer